packages/probing-frontend/src/pages/HomePage/components/MonitoringTable.tsx

```diff
@@ -0,0 +1,139 @@
+import { ButtonNaked } from '@pagopa/mui-italia'
+import { useTranslation } from 'react-i18next'
+import RefreshIcon from '@mui/icons-material/Refresh'
+import { MonitoringTableRow } from './MonitoringTableRow'
+import { MonitoringQueries } from '@/api/monitoring/monitoring.hooks'
+import { Skeleton } from '@mui/material'
+import { useLoadingOverlay } from '@/stores'
+import type { TFunction } from 'i18next'
+
+const headLabels = (t: TFunction<'common', 'table'>): Array<string> => {
+  return [
+    t('eServiceName'),
+    t('version'),
+    t('producer'),
+    t('eServiceStatus'),
+    t('lastDetectionDate'),
+    '',
+  ]
+}
+
+export const MonitoringTable = () => {
+  const { t } = useTranslation('common', { keyPrefix: 'table' })
+  const { paginationParams, paginationProps, getTotalPageCount } = usePagination({ limit: 10 })
+  const { showOverlay, hideOverlay } = useLoadingOverlay()
+
+  const [producersAutocompleteTextInput, setProducersAutocompleteTextInput] =
+    useAutocompleteTextInput()
+
+  const { data: producerOptions } = MonitoringQueries.useGetProducersList(
+    { offset: 0, limit: 20, producerName: producersAutocompleteTextInput },
+    { suspense: false }
+  )
+
+  const { filtersParams, ...handlers } = useFilters([
+    {
+      name: 'eserviceName',
+      type: 'freetext',
+      label: t('serviceNameFilter'),
+    },
+    {
+      name: 'producerName',
+      type: 'autocomplete-single',
+      label: t('serviceProducerFilter'),
+      onTextInputChange: setProducersAutocompleteTextInput,
+      options: producerOptions ?? [],
+    },
+    {
+      name: 'versionNumber',
+      type: 'numeric',
+      label: t('serviceVersionFilter'),
+    },
+    {
+      name: 'state',
+      type: 'autocomplete-multiple',
+      label: t('serviceStateFilter'),
+      options: [
+        { value: 'ONLINE', label: 'online' },
+        { value: 'OFFLINE', label: 'offline' },
+        { value: 'N_D', label: 'n/d' },
+      ],
+    },
+  ])
+
+  const params = {
+    ...paginationParams,
+    ...filtersParams,
+  }
+
+  const {
+    data: eservices,
+    refetch,
+    isInitialLoading,
+  } = MonitoringQueries.useGetList(params, { suspense: false })
+
+  const handleRefetch = async () => {
+    showOverlay(t('loading'))
+    // We want show the loading overlay for at least 1 second, to avoid flickering
+    await Promise.all([refetch(), new Promise((resolve) => setTimeout(resolve, 1000))])
+    hideOverlay()
+  }
+
+  return (
+    <>
+      <Filters
+        {...handlers}
+        rightContent={
+          <ButtonNaked
+            color="primary"
+            onClick={handleRefetch}
+            size="small"
+            startIcon={<RefreshIcon />}
+          >
+            {t('refresh')}
+          </ButtonNaked>
+        }
+      />
+
+      {isInitialLoading ? (
+        <TableSkeleton />
+      ) : (
+        <Table headLabels={headLabels(t)} isEmpty={!eservices || eservices?.content?.length === 0}>
```

Comment:
`noDataLabel` props must be given to the `Table` component in order to show the correct message when there are no results